program.html

```diff
@@ -37,156 +40,168 @@
                         <li class="item"><a href="#copic-manga">COPIC-, Manga-Workshop</a></li>
                     </ul>
-                    </li>
+                </li>
 
-                    <li class="item"><a href="#week2">06.07.20 - 10.07.20</a>
-                        <ul>
-                                <li class="item"><a href="#painting">Painting</a></li>
-                                <li class="item"><a href="#painting-masterclass-oil">Oil painting masterclass</a></li>
-                                <li class="item"><a href="#copic-manga2">COPIC-, Manga-Workshop</a></li>
-                            </ul>
-                    </li>
+                <li class="item"><a href="#week2">06.07.20 - 10.07.20</a>
+                    <ul>
+                        <li class="item"><a href="#painting">Painting</a></li>
+                        <li class="item"><a href="#painting-masterclass-oil">Oil painting masterclass</a></li>
+                        <li class="item"><a href="#copic-manga2">COPIC-, Manga-Workshop</a></li>
+                    </ul>
+                </li>
 
-                    <li class="item"><a href="#week3">13.07.20 - 17.07.20</a></li>
-                    <li class="item"><a href="#week4">20.07.20 - 24.07.20</a></li>
-                    <li class="item"><a href="#week5">27.07.20 - 31.07.20</a></li>
-                    <li class="item"><a href="#week6">03.08.20 - 07.08.20</a></li>
-                </ul>
-            </div>
-            <div class="content-right">
-                <div class="item-container">
-                        <div class="about-item">
-                                <div class="text-part">
-                                        <h2 id="week1">Week 1</h2>
-                                        <p>29.06.20 - 03.07.20</p>
-                                    </div>
-                                </div>
-                                <hr>
-                    <div class="about-item">
-                        <div class="text-part">
-                             <h3 id="drawing1">Drawing course</h3>
-                                <p><b>Theme:</b> Drawing and painting course includes basic knowledge about light, shadow, perspective. With these principles you will learn how to draw people and simple objects. <br>
-                                <b>For whom:</b> Adults and teens from ~12 years <br>
-                                <b>Time:</b> Monday to Friday from 9:30 to 16:00 <br>
-                                <b>Price:</b> 248,00 Euro (incl. VAT and materials).*<br>
-                                <b>Place:</b> Studio 1st floor<br>
-                            </p>
-                        </div>
-                        <div class="image-part">
-                            <img src="assets/program-drawing.jpg" alt="Drawing course" width="100%">
-                        </div>
+                <li class="item"><a href="#week3">13.07.20 - 17.07.20</a></li>
```

Comment:
I think in general you would want to have some content management system connected to your web site. Because in current setup you would need to change this html page each time courses should be updated/added/removed.
I never used it but looks like you can use wordpress for that - store and update content there, and grab this content via REST api https://developer.wordpress.org/rest-api/

Reply:
Yes, thank you for the suggestion. The main school website is now made on Wordpress but they are still changing everything by hand. It's a matter of their business organization, you know, but if they decide to switch to the API model of grabbing content for the courses it would be a nice thing to invent.